### DISCUSSION TITLE
Covalent resource scheduler?

### DISCUSSION THREAD

**Discussion by ealigholi** [AgnostiqHQ/covalent#1805] [General]
does covalent have resource scheduler?

**Comment by santoshkumarradha** [comment]
Hey @ealigholi , we do not have our own scheduler, but instead we tap into different schedulers like SLURM, Batch etc... Although we dont have a scheduler, we do have our own queue system, by default the scheduler we use when no executor is specified is Dask.

**Comment by ealigholi** [comment]
does it have cache management?

**Reply by santoshkumarradha** [reply]
If by cache management you mean caching and reusing workflow and subparts of workflow, then yes. All meta data about workflows are cached and reused to its maximum capability. If you have a specific setup in mind, we can work to see if it fits in.
